app/controllers/sessions_controller.rb

```diff
@@ -12,7 +14,7 @@ def create
           log_in user
           redirect_back_or user
         else
-          flash[:error] = 'Your account has not been approved by an administrator'
+          flash.now[:error] = 'Your account has not been approved by an administrator'
           render 'new'
```

Comment:
should be :danger, or something else. Error doesn't have any background color to it.